translate/translate_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2019 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translate
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/coreos/ignition/v2/config/translate/tests/pkga"
```

Comment:
Should be using the `fcct` paths rather than the `Ignition` paths.

Reply:
This gets fixed in the next commit. I wanted to copy-over to be verbatim so it's easier to see what changed exactly.